Code-the-Dream-School/hh-team4-back
# Update med model

## Description

Updated user model to include: 
Name 
Generic Name 
Lot (instead of BatchCode) 
expirationDate
Class (instead of type) 
Store
NDC number
createdBy
threshold
quantity

I also incorporated the logic to display the user name in the createdBy field.

## Related Issue
Front requested said changes.

## Update Screenshots

### Before
![old med model](https://github.com/user-attachments/assets/70fa4e08-973b-4fe9-a6ad-9e80308eb883)
### After
![New med model](https://github.com/user-attachments/assets/d94dcf62-c307-46e7-8d06-4bc9ce4011fb)
## Testing Instructions
Create a new item in Postman. 

Note, Will need to signUp/Login in order to get bearer token , 
Bearer token is needed in auth tab of Create new med request. 

Note: I recreated Auth on just the create route, Need to delete in conflict resolving and keep auth routes from UserAuth branch.

Note: I had some issues with mongoDB keeping the batchcode label from the old model , and used mongoosh CLI to check the database and delete the field, and then add the new. I assume it will stay that way...
